ecosystem/sep-0039.md

```diff
@@ -0,0 +1,193 @@
+## Preamble
+
+```
+SEP: 0039
+Title: On-Chain Storage of Non-Fungible Assets
```

Comment:
A little while ago @lydiat highlighted the benefits in having succinct SEP names.

Following the pattern of SEP-11 Txrep about:
```suggestion
Title: On-Chain NFTs: Storage of Non-Fungible Assets On-Chain
```

Reply:
I'm of the opinion we should keep this sep laser focused on NFTs. If some other sep in the future wants to adopt our methods for other arbitrary data storage, fine, but to keep this discussion focused we need to stay on target with the task at hand which is delivering consensus on how NFTs should be stored on-chain